tests/llm/test_vertexai/test_simple_types.py

```diff
@@ -24,22 +24,6 @@ def test_literal(model, mode):
     assert response in ["1231", "212", "331"]
 
 
-@pytest.mark.parametrize("model, mode", product(models, modes))
```

Comment:
We can see that the `Union` type json dump produces a schema which has `anyOf` in it. This isn't supported by Gemini and is indicated in our docs here

> https://python.useinstructor.com/hub/vertexai/#optional-anyof

```python
from pydantic import BaseModel
from typing import Union


class User(BaseModel):
    property: Union[int, str]


print(User.model_json_schema())
# {
#     "properties": {
#         "property": {
#             "anyOf": [{"type": "integer"}, {"type": "string"}],
#             "title": "Property",
#         }
#     },
#     "required": ["property"],
#     "title": "User",
#     "type": "object",
# }

```